pkg/api/handlers/types.go

```diff
@@ -128,11 +128,16 @@ type CreateContainerConfig struct {
 	NetworkingConfig dockerNetwork.NetworkingConfig
 }
 
+// swagger:model VolumeCreate
 type VolumeCreateConfig struct {
-	Name   string            `json:"name"`
-	Driver string            `schema:"driver"`
-	Label  map[string]string `schema:"label"`
-	Opts   map[string]string `schema:"opts"`
+	// New volume's name. Can be left blank
+	Name string `json:"name"`
+	// Volume driver to use
+	Driver string `schema:"driver"`
```

Comment:
FYI we don't support setting this in Libpod right now, will throw errors if it's passed. That's probably OK.